src/ripple/core/impl/Config.cpp

```diff
@@ -485,24 +485,29 @@ Config::loadFromString(std::string const& fileContents)
     {
         using namespace std::chrono;
         boost::regex const re(
-            "^\\s*(\\d+)\\s*(seconds|minutes|hours|days|weeks)\\s*(\\s+.*)?$");
+            "^\\s*(\\d+)\\s*(minutes|hours|days|weeks)\\s*(\\s+.*)?$");
         boost::smatch match;
         if (!boost::regex_match(strTemp, match, re))
             Throw<std::runtime_error>(
                 "Invalid " SECTION_AMENDMENT_MAJORITY_TIME
-                ", must be: [0-9]+ [seconds|minutes|hours|days|weeks]");
+                ", must be: [0-9]+ [minutes|hours|days|weeks]");
+
         std::uint32_t duration =
             beast::lexicalCastThrow<std::uint32_t>(match[1].str());
-        if (boost::iequals(match[2], "seconds"))
-            AMENDMENT_MAJORITY_TIME = seconds(duration);
-        else if (boost::iequals(match[2], "minutes"))
+
+        if (boost::iequals(match[2], "minutes"))
             AMENDMENT_MAJORITY_TIME = minutes(duration);
         else if (boost::iequals(match[2], "hours"))
             AMENDMENT_MAJORITY_TIME = hours(duration);
         else if (boost::iequals(match[2], "days"))
             AMENDMENT_MAJORITY_TIME = days(duration);
         else if (boost::iequals(match[2], "weeks"))
             AMENDMENT_MAJORITY_TIME = weeks(duration);
+
+        if (AMENDMENT_MAJORITY_TIME < defaultAmendmentMajorityTime)
```

Comment:
I think this check is wrong. It will prevent anything less than 2 weeks. I'd simply hard-code `std::chrono::minutes(15)` here.

Reply:
Right!!!